Zend/tests/bug75573.phpt

```diff
@@ -31,10 +27,6 @@ class A
 
 class B extends A
 {
-	function initialize($properties = array())
-	{
-		parent::initialize($properties);
-	}
```

Comment:
See this [comment on bugs.php.net](https://bugs.php.net/bug.php?id=75573#1532456722):

> Did the unused initialize methods have some bearing on this bug?
> Since `A` does not have a parent it would fatal if it were ever called. I'm adding compile-time warnings to uses of `parent` when there isn't one and ran across this bug, but since it's "fixed" I can't tell if removing the unused methods has any bearing.

Reply:
Just don't make unrelated "improvements" in PHPT tests, because they look suspicious, and require reviewers to verify them. Better, commit these chunks separately.

Reply:
These are not unrelated. The unused method `A::initalize` has erroneous `parent::` usage, which this includes. I know you said you would have preferred that to be a different patch, but the code that deals with `parent` is needed for resolving variance.

Reply:
It would be better to add compile-time warning. This would clearly explain the change in behavior, and its correctness.